velox/connectors/Connector.h

```diff
@@ -135,29 +137,35 @@ CommitStrategy stringToCommitStrategy(const std::string& strategy);
 /// to be thread-safe.
 class DataSink {
  public:
+  struct Stats {
+    uint64_t completedBytes{0};
+    uint32_t numWrittenFiles{0};
+    common::SpillStats spillStats;
+    /// Contains metadata for each written table partition.
+    std::vector<std::string> partitions;
```

Comment:
Connector is a generic concept, but partitions are Hive-specific. Would it be possible to keep Connector API generic?

Reply:
Then I rename it to writtenTableMetadata so let connector to pass customized metadata?

Reply:
I keep the old close to return the metadata things and keep the stats separate per offline discussion with @mbasmanova Thanks!